pkg/hs/urbit-king/lib/Urbit/Arvo/Event.hs

```diff
@@ -218,9 +218,7 @@ instance Show Entropy where
 data ArvoEv
     = ArvoEvWhom ()   Ship
     | ArvoEvWack ()   Entropy
-    | ArvoEvWarn Path Noun
     | ArvoEvCrud Path Noun
-    | ArvoEvVeer Atom Noun
   deriving (Eq, Ord, Show)
```

Comment:
This should be `$%(waif wasp wisp)`, where all three are defined in `arvo.hoon`:

```
+$  waif
  ::  %trim: trim state, spam to all
  ::  %what: update from files
  ::  %whey: produce $mass                    :: XX remove, scry
  ::  %verb: toggle laconicity
  ::
  $%  [%trim p=@ud]
      [%what p=(list (pair path (cask)))]
      [%whey ~]
      [%verb p=(unit ?)]
  ==
+$  wasp
  ::  %crud: reroute $ovum with $goof
  ::  %wack: iterate entropy
  ::  %wyrd: check/record runtime kelvin stack
  ::
  $%  [%crud =goof =ovum]
      [%wack p=@uvJ]
      [%wyrd p=vere]
  ==
+$  wisp
  $%  $>(?(%verb %what) waif)
      $>(?(%wack %wyrd) wasp)
      [%whom p=ship]
  ==
```

All of `$waif` and `$>(%wyrd wasp)` (version negotiation event) are missing; `%crud` is wrong.

Reply:
Elliot added Wyrd, and I've merged his changes in so this should no longer be an issue. How should waif be handled? What processing needs to be done with them, if any?

Reply:
I've merged in Elliot's work so we have Wyrd now. As for waif, what if anything should I do with these events? It seems like I can fail to support whey and verb without issue, but what are the semantics of trim and what?

Reply:
They're events, not effects, so you'd be generating them, not implementing their semantics. You only *need* them if you'll be a) injecting them or b) parsing events produced elsewhere.